docs/dapps/nfts/preparing-nft-files.md

```diff
@@ -0,0 +1,88 @@
+# Preparing NFT Files for an ERC721 Smart Contract
+
+The first step of setting up an NFT smart contract is having your NFT files ready to use. In this example, the files will get uploaded to [Pinata](https://www.pinata.cloud/), a pinning service that prevents files from being garbage collected on IPFS. If you do not already have an account, please create one.
+
+## Preparing the images
+
+This tutorial will create only 1 NFT, however, if you are interested in creating more you are more than welcome to do so. The image I am using is linked here if you would like to use it.
+
+![Original NFT Photo](preparing-nft-files/1-original.jpeg)
+
+Place your image file in a folder on your computer. Name this image `0`, so it will be the first image pulled from the smart contract. It will be the first (and only) NFT in this collection, however, if you are adding more images you would continue naming them in sequential numeric order. You will upload this folder to Pinata once your images are organized and named correctly.
+
+:::info
+NOTE: Some projects start file names with `0`, and others with `1`. That choice will need to be consistent with the smart contract code. To be consistent with [this ERC721 tutorial](intro-to-erc721s.md), we will name this file `0`.
+:::
+
+![Image Folder](preparing-nft-files/2-image-folder.png)
+
+After you log into Pinata, you will see your dashboard. You will see the upload button on the left. Click `Upload`, and then `Folder`.
```

Comment:
Consider using contractions to make the text read more casually.

Ex: "you'll see your dashboard..." instead of "you will see your dashboard..." and "You'll see the upload button..." instead of "You will see the upload button..."